airflow-core/src/airflow/api_fastapi/core_api/routes/public/dag_parsing.py

```diff
@@ -66,3 +71,63 @@ def reparse_dag_file(
 
     parsing_request = DagPriorityParsingRequest(fileloc=path)
     session.add(parsing_request)
+
+
+@dag_parsing_router.post(
+    "/manage/reserialize",
+    responses=create_openapi_http_exception_doc(
+        [
+            status.HTTP_400_BAD_REQUEST,
+            status.HTTP_404_NOT_FOUND,
+            status.HTTP_409_CONFLICT,
+            status.HTTP_500_INTERNAL_SERVER_ERROR,
```

Comment:
```suggestion
```

This shouldn't be doncumented. That's unexpected and any endpoint can throw unexpected 500.